ldap-api/src/test/java/ca/bc/gov/hlth/ldapapi/service/LdapServiceTest.java

```diff
@@ -27,64 +27,15 @@ public static void initialSetup() {
     }
 
     @Test
-    public void testCreateReturnMessage_authenticatedFalse() {
```

Comment:
I removed these tests because they no longer compile and I don't think they're worth fixing. The `createUserMessage` method just copies some values between objects, there's no interesting business logic.

(If we want to spend time on testing, I recommend we add integration tests to actually call the REST endpoint. We'd need to refactor a bit to mock-out the calls that depend on LDAP.)

Reply:
You can also remove the `userName`, `attributesWithRole`, and `attributesWithoutRole` variables as they are no longer used. Doing this would also create unused imports for `Attributes` and `BasicAttributes` that could be removed.